### PR TITLE
refactor: update readme for the project

### PR DESCRIPTION
**Pull request by thenishantsapkota**

### CODE REVIEW

**Comment by thenishantsapkota** [comment]
> Maybe we would want to add `CONTRIBUTING.md` file as well, and put the file format over there instead?
> 
> I feel it's better if we keep the simple to end users who might be referring to it and use it. For development purposes we can always have `CONTRIBUTING.md` and more details on it.

That would be a good way to go!
I'll include a `CONTRIBUTING.md` in this PR.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ "location": "Fully remote",
+ "website": "https://anawesomewebsite.com",
+ "socials": {
+   "facebook": "https://facebook.com",
+   "instagram": "https://instagram.com",
+   "github": "https://github.com",
+   .
+   .
+   .
+   // Social media handles will go here
```
Comment (TheLearneer) [inline CONTRIBUTING.md:18]:
```suggestion
```
I believe it's best to keep the JSON format valid

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   }
+},
+.
+.
+.
+// Several Companies here...
```
Comment (TheLearneer) [inline CONTRIBUTING.md:24]:
```suggestion
```
I believe it's best to keep the JSON format valid

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+]
+```
+
```
Comment (TheLearneer) [inline CONTRIBUTING.md:27]: Add a mention that keys not provided here can be added as well, but it's best if it's discussed with the team through the discussions tab or our discord server 🙏

**Review by TheLearneer — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (thenishantsapkota) [reply CONTRIBUTING.md:19]: That should be discussed with the team so lets not do that right now!

Comment (TheLearneer) [reply CONTRIBUTING.md:19]: That sounds good.
But then maybe we should mention all the keys as discussed in #1 ??

Comment (thenishantsapkota) [reply CONTRIBUTING.md:19]: > That sounds good. But then maybe we should mention all the keys as discussed in #1 ??

I'll list those right away!

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**